src/core/logical_layers/utils/layersOrder/layersOrder.ts

```diff
@@ -43,76 +63,120 @@ export class LayersOrderManager {
       });
     });
   }
+  _initForTests(
+    layersSettings: Map<string, AsyncState<LayerSettings, Error>>,
+    layersParentsIds: Map<string, string>,
+  ) {
+    this._layersSettings = layersSettings;
+    this._layersParentsIds = layersParentsIds;
+  }
```

Comment:
[Don't cheat](https://news.drom.ru/volkswagen-36397.html?page=2).

Correct way to do it:

````ts
 init(map: MapLibre, layersSettingsAtom = layersSettingsAtom, generatedMapboxLayersParents = generatedMapboxLayersParents) {
   this.layersSettingsAtom = layersSettingsAtom;
   this.generatedMapboxLayersParents = generatedMapboxLayersParents;
 }
 ```